mirdata/beatles.py

```diff
@@ -87,23 +127,59 @@ def download(data_home=None, force_overwrite=False):
 
 
 def exists(data_home=None):
+    """Return if Beatles dataset folder exists
+
+    Args:
+        data_home (str): Local home path that the dataset is being stored.
+
+    Returns:
+        (bool): True if Beatles dataset folder exists
+
+    """
     save_path = utils.get_save_path(data_home)
     dataset_path = os.path.join(save_path, DATASET_DIR)
     return os.path.exists(dataset_path)
 
 
 def validate(dataset_path, data_home=None):
+    """Validate if the stored dataset is a valid version
+
+    Args:
+        dataset_path (str): Beatles dataset local path
+        data_home (str): Local home path that the dataset is being stored
+
+    Returns:
+        missing_files (list): List of file paths that are in the dataset index
+            but missing locally
+        invalid_checksums (list): List of file paths that file exists in the dataset
+            index but has a different checksum compare to the reference checksum
+
+    """
     missing_files, invalid_checksums = utils.validator(
         INDEX, data_home, dataset_path
     )
     return missing_files, invalid_checksums
 
 
 def track_ids():
+    """Return track ids
+
+    Returns:
+        (list): A list of track ids
+    """
     return list(INDEX.keys())
 
 
 def load(data_home=None):
+    """Load Beatles dataset
+
+    Args:
+        data_home (str): Local home path that the dataset is being stored.
```

Comment:
Location on disk where the dataset is stored. If `None`, looks for the data in the default directory, `~/mir_datasets`.